third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -41,6 +41,7 @@ const TEST_CONFIGS: &[TestConfig] = &[
         runner: |p| run(p, get_config_by_name("optimize")),
         experiments: &[
             (Experiment::OPTIMIZE, true),
+            (Experiment::CFG_SIMPLIFICATION, true),
```

Comment:
See note regarding derivation in `testsuite.rs`.

Reply:
done